quark.h

```diff
@@ -275,6 +275,8 @@ struct raw_sock_conn {
 	struct quark_sockaddr	local;
 	struct quark_sockaddr	remote;
 	enum sock_conn		conn;
+	uint64_t	bytes_received;
+	uint64_t	bytes_sent;
```

Comment:
```suggestion
	u64	bytes_received;
	u64	bytes_sent;
```

Reply:
forced pushed the change: [4187467](https://github.com/elastic/quark/pull/250/commits/41874678e53e0fafa02b96517324652843ecea4f)